projects/view/client/projectProfile/projectProfile.html

```diff
@@ -5,50 +5,70 @@
-        <label>Launched in: </label><span>{{startupDateFormatted}}</span><br/>
-        <label>Based in: </label><span>{{postalAddress.country}}</span><br/>
-        <label>Website: </label>
-        {{# if hasMultipleWebsites }}
-          {{# each websites }}
-            <span class="label label-default {{typeToClass}}"><a href="{{url}}">{{name}}</a></span>
-          {{/ each }}
-        {{else}}
-          {{# each websites }}
-            <span class="label label-default"><a href="{{url}}">{{url}}</a></span>
-          {{/ each }}
-        {{/ if}}
+        <div id="project-data">
+          <label>Progress: </label>
 
-        <br/>
-        <label>Application</label>
-        {{# each platform in targetPlatforms }}
-          <span class="label label-default">{{platform}}</span>
-        {{/ each }}
-        <br/>
+          <div class="progress">
+            <div class="progress-bar" role="progressbar" aria-valuenow="{{calculateProgress}}" aria-valuemin="0" aria-valuemax="100" style="width: {{calculateProgress}}%;">
+              <span class="sr-only">{{calculateProgress}}% Complete</span>
+            </div>
+          </div>
 
-        <label>Social</label>
-        {{# each socialLinks }}
-          <span class="label label-default {{typeToClass}}"><a href="{{url}}">{{name}}</a></span>
-        {{/ each }}
-        <br/>
+          <ul id="project-list">
+            <li><label>Launched in </label><span> {{startupDateFormatted}}</span></li>
+            <li><label>Categorys </label>
+              {{# each category in problemCategories }}
+                <a href="categories/{{category}}" class="project-data-link">{{category}}</a>
+              {{/ each }}
+            </li>
+            <!-- How do we link country issues, what is the route? -->
+            <li><label>Based in  </label><a href="country/{{postalAddress.country}}" class="project-data-link"> {{postalAddress.country}}</a></li>
+            <li><label>Website </label>
+              {{# if hasMultipleWebsites }}
+                {{# each websites }}
+                  <a href="{{url}}" class="project-data-link">{{name}}</a>
+                {{/ each }}
+              {{else}}
+                {{# each websites }}
+                  <a href="{{url}}" class="project-data-link">{{url}}</a>
+                {{/ each }}
+              {{/ if}}
+            </li>
+            <li><label>Application </label>
+              {{# each platform in targetPlatforms }}
+                <a href="type/{{platform}}" class="project-data-link">{{platform}}</a>
+              {{/ each }}
+            </li>
+            <li><label>Social </label>
+              {{# each socialLinks }}
+                <a href="{{url}}" class="project-data-link">{{name}}</a>
+              {{/ each }}
+            </li>
+          </ul>
 
+        </div>
+        <div id="project-description">{{ description }}</div>
+        <div id="project-buttons">
+          <!-- TODO: Round corners on buttons. -->
```

Comment:
[Bootstrap buttons](http://getbootstrap.com/components/#btn-groups) have rounded corners. Is this comment still relevant?